src/main/java/com/acoustic/salarycalculator/menu/SalaryCalculatorMenu.java

```diff
@@ -1,27 +1,27 @@
-package org.example.enumApproch.menu;
+package com.acoustic.salarycalculator.menu;
 
 import java.math.BigDecimal;
 import java.util.Arrays;
 import java.util.Scanner;
 
-import org.example.enumApproch.enums.JobsCategory;
-import org.example.enumApproch.print.PrinterSalaryCalculator;
+import com.acoustic.salarycalculator.jobscategories.JobsCategory;
+import com.acoustic.salarycalculator.printer.SalaryCalculatorPrinter;
 
 
-public class MenuSalaryCalculator {
+public class SalaryCalculatorMenu {
 
     private final Scanner scanner;
-    private final PrinterSalaryCalculator printerCalculator;
+    private final SalaryCalculatorPrinter printerCalculator;
 
-    public MenuSalaryCalculator() {
+    public SalaryCalculatorMenu() {
         this.scanner = new Scanner(System.in);
-        this.printerCalculator = new PrinterSalaryCalculator();
+        this.printerCalculator = new SalaryCalculatorPrinter();
 
     }
```

Comment:
method jobTitleValidator -> there are 3 nested loops, try to avodid nested loops, this complicate your program, and time for operations, 
https://stackoverflow.com/questions/362059/what-is-the-big-o-of-a-nested-loop-where-number-of-iterations-in-the-inner-loop

Reply:
you could ask about department, then about job , ask only for department number, not whole name, the same for job, this will be simplest